ablySpec/TestUtilities.swift

```diff
@@ -167,5 +222,5 @@ class MockHTTPExecutor: ARTHTTPExecutor {
     func executeRequest(request: NSMutableURLRequest!, callback: ((NSHTTPURLResponse!, NSData!, NSError!) -> Void)!) {
         self.requests.append(request)
         self.executor.executeRequest(request, callback: callback)
-    }
+    }    
```

Comment:
Strange. I have that option activated on Xcode: `Automatically trim trailing whitespace`. Maybe a bug 😅